From: Michael Biebl <biebl@debian.org>
Date: Tue, 10 Mar 2015 22:55:22 +0100
Subject: Don't explicitly link tcpflood against -lgcrypt

While older GnuTLS versions do use gcrypt, newer version no longer do.
We rely on GnuTLS to pull that library dependency in.
---
 tests/Makefile.am | 3 ---
 1 file changed, 3 deletions(-)

diff --git a/tests/Makefile.am b/tests/Makefile.am
index 75ed68c..ee544e8 100644
--- a/tests/Makefile.am
+++ b/tests/Makefile.am
@@ -1839,9 +1839,6 @@ tcpflood_SOURCES = tcpflood.c
 tcpflood_CFLAGS = $(PTHREADS_CFLAGS) $(RELP_CFLAGS) $(GNUTLS_CFLAGS)
 tcpflood_CPPFLAGS = $(PTHREADS_CFLAGS) $(RELP_CFLAGS) $(GNUTLS_CFLAGS)
 tcpflood_LDADD = $(SOL_LIBS) $(PTHREADS_LIBS) $(RELP_LIBS) $(GNUTLS_LIBS)
-if ENABLE_GNUTLS
-tcpflood_LDADD += -lgcrypt
-endif
 
 minitcpsrv_SOURCES = minitcpsrvr.c
 minitcpsrv_LDADD = $(SOL_LIBS)
